seed/challenges/05-apis-and-microservices/api-and-microservice-projects.json

```diff
@@ -148,11 +148,11 @@
           "testString": ""
         },
         {
-          "text": "I can add an exercise to any user by posting form data userId(_id), description, duration, and optionally date to /api/exercise/add. If no date supplied it will use current date. Returned will the the user object with also with the exercise fields added.",
+          "text": "I can add an exercise to any user by posting form data userId(_id), description, duration, and optionally date to /api/exercise/add. If no date supplied it will use current date. App will return the user object with also with the exercise fields added.",
```

Comment:
`App will return the user object with also with the exercise fields added."` 

should be:

`App will return the user object with the exercise fields added."`

Reply:
There. I fixed it. thank you for review. I didn't notice that one. ha ha.